pkgs/games/quakespasm/vulkan.nix

```
@@ -53,7 +55,7 @@ stdenv.mkDerivation rec {
       specialization constants, CPU/GPU parallelism and memory pooling.
     '';
 
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ ];
+    platforms = platforms.linux ++ [ "aarch64-darwin" ];
```

vkQuake also builds and runs on x86_64-darwin.

```suggestion
    platforms = platforms.linux ++ platforms.darwin;
```

---

Oh, nice. Pushed accordingly.